src/database.ts

```diff
@@ -0,0 +1,15 @@
+import * as mongoose from "mongoose";
+
+class Database {
+  public db_connect(mongoUrl: string | undefined, mongoDb: string | undefined) {
```

Comment:
public connect(url: string, database: string)

Reply:
I've modified the code.

Reply:
* to use public connect(url: string, database: string)

Reply:
I'm getting this error now though when linting it:

**src/server.ts(11,11): error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.**